coreos-base/oem-packet/files/phone-home.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+curl -s -o /tmp/jq http://images.packet.net/jq
```

Comment:
This is terrifying. Either we should add jq to the image or perhaps ship a little go app in the OEM partition that does this instead of a shell script.